core/src/test/java/org/elasticsearch/index/mapper/LegacyStringMappingTests.java

```diff
@@ -731,9 +731,8 @@ public void testNonAnalyzedFieldPositionIncrement() throws IOException {
                 .field("position_increment_gap", 10)
                 .endObject().endObject().endObject().endObject().string();
 
-            IllegalArgumentException e = expectThrows(IllegalArgumentException.class,
-                () -> parser.parse("type", new CompressedXContent(mapping)));
-            assertEquals("Cannot set position_increment_gap on field [field] without positions enabled", e.getMessage());
+            // allowed in index created before 5.0
+            DocumentMapper defaultMapper = parser.parse("type", new CompressedXContent(mapping));
```

Comment:
The point is now just to see there is no error? Then there is no reason to create a local var if there are no assertions done on it?